src/services/LogFileManager/index.ts

```diff
@@ -301,7 +310,6 @@ class LogFileManager {
             "" :
             "i";
         const queryRegex = new RegExp(regexPattern, regexFlags);
-
         this.#queryChunkAndScheduleNext(this.#queryId, 0, queryRegex);
```

Comment:
I feel this statement acts as a return statement and we want to put an empty line before that.
```suggestion

        this.#queryChunkAndScheduleNext(this.#queryId, 0, queryRegex);
```